css/cssom-1/cssom-ruleOrder.html

```diff
@@ -1,7 +1,8 @@
 <!doctype html>
 <head>
-  <title>CSS OM: CSS Values</title>
+  <title>CSS OM: CSS Rule Order</title>
   <link rel="author" title="Divya Manian" href="mailto:manian@adobe.com">
+  <link rel="author" title="Melanie Richards" href="mailto:Melanie.Richards@microsoft.com">
```

Comment:
No need to maintain author links (we have them all in git history, except in the case of old contributors who didn't have VCS access when the CSS testsuite was in CVS/SVN), and we have Divya as the original author in the file's history, so you're just as well off deleting it.